src/managed/OpenLiveWriter.PostEditor/PostHtmlEditing/ImageEditing/ImageEditingPropertyHandler.cs

```diff
@@ -93,8 +95,16 @@ public static ImagePropertiesInfo GetImagePropertiesInfo(IHTMLImgElement imgElem
                     imgHtmlElement.setAttribute("height", oldHeight, 0);
                 if (!String.IsNullOrEmpty(oldWidth))
                     imgHtmlElement.setAttribute("width", oldWidth, 0);
-
-                info = new ImagePropertiesInfo(new Uri(imgSrc), new Size(width, height), remoteImageDecoratorsList);
+                try
+                {
+                    info = new ImagePropertiesInfo(new Uri(imgSrc), new Size(width, height), remoteImageDecoratorsList);
+                }
+                catch (UriFormatException)
```

Comment:
I think it's better to use Uri.TryCreate instead of using try catch to detect invalid uris.